### DISCUSSION TITLE
Woocommerce Checkout Block Suggestion

### DISCUSSION THREAD

**Discussion by MTraveller** [woocommerce/woocommerce#45044] [WooCommerce, Blocks, and Site Editing]
## Is your feature request related to a problem? Please describe.
I get frustrated when I go to a shop and see something I like and decide to buy. When arriving to the checkout page I see an old, messy page with lot's of inputs screaming - run away!

## Describe the solution you'd like
Now the new woocommerce checkout block is awesome, a big leap towards greatness, but it is not there yet. I am dreaming of bullets;

1. Contact information
2. Billing address
3. Payment method

Play nice together in a smooth flow, like the water on a river. All is flowing all is great ;)

Imagine you arrive at the checkout page. You see the 1. Contact information like a accordion and is the only one open then automatically closes by moving to the next bullet/ accordion - 2. billing address the flow continues, next - 3. payment method opens while 2. billing address closes all in smooth transition.

That's a good looking checkout.

And those hooks to hook to completes it all moving to a more sunny place where peace and love prevails :)

## Describe alternatives you've considered
Hope

## Additional context
Advance Expand Progress Evolve

**Comment by nerrad** [comment]
HI there @MTraveller  👋🏻

Thanks so much for taking the time to leave some feedback - much appreciated! We make note of all feedback we receive and we also close issues with the feedback as active issues are used for bugs or feature/enhancements. It does not mean that we don't note or review the feedback. Closed issues can also still be commented on.
